pkg/reconciler/v1alpha1/configuration/resources/build.go

```diff
@@ -53,7 +53,7 @@ func MakeBuild(config *v1alpha1.Configuration) *unstructured.Unstructured {
 	u.SetLabels(l)
 
 	u.SetNamespace(config.Namespace)
-	u.SetName(names.DeprecatedBuild(config))
+	u.SetGenerateName(fmt.Sprintf("%s-", config.Name))
```

Comment:
```suggestion
	u.SetGenerateName(config.Name+"-")
```
Seems faster and easier to read?